libraries/core_libs/network/src/tarcap/packets_handlers/pbft_sync_packet_handler.cpp

```diff
@@ -67,6 +67,10 @@ void PbftSyncPacketHandler::process(const PacketData &packet_data, const std::sh
     }
   }
 
+  if (!pbft_mgr_->checkBlockWeight(sync_block)) {
+    throw MaliciousPeerException("Block summary gas amount is bigger then limit");
+  }
```

Comment:
The check has issue when there are more DAG blocks in single level.
I think we cannot check here anymore. Because when node just receive synced PBFT block, node has not have DAG yet. That need check just before adding the block to PBFT chain